src/views/BookmarksView/BookmarksView.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { toast } from "react-toastify";
+import { getMyBookmarkedArticlesAction } from "../../actions/getMyBookmarkedArticlesAction";
+import BookmarkedList from "../../components/BookmarksList/BookmarksList";
+
+export class BookmarksView extends Component {
+  /**
+   *
+   * @param {} props - props passed into the JSX initialisation of the component
+   */
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  componentDidMount() {
+    const token = window.localStorage.getItem("token");
+    const { getMyBookmarkedArticlesAction } = this.props;
+    getMyBookmarkedArticlesAction(token);
+  }
+  /**
+   * Receives props after action is dispatched.
+   * @param {} nextProps - properties object that is updated when an
+   * action is dispatched and mapStateToProps below is invoked.
+   */
+  componentWillReceiveProps(nextProps) {
+    const bookmarks = nextProps.bookmarks;
+    const { history } = this.props;
+    if (bookmarks) {
+      if (bookmarks.detail) {
+        this.setState({ toBeRendered: bookmarks.detail, noData: true });
```

Comment:
Shouldn't you have default values for this state properties? @bibangamba 

Otherwise implementation is well done

Reply:
@JaymesKat, not sure I understand. Because I get `bookmarks` from `nextProps` so I do the check to prevent a run error if/when `componentWillReceiveProps` is invoked but `bookmarks` is not in that `nextProps` object.

Reply:
Just finished doing this and a slight styles change. Please check out the PR screenshots for the updated version.